.travis.yml

```
@@ -77,6 +77,22 @@ jobs:
       script:
         - NODE=$(which node) make lint lint-py
 
+    - name: "Docs"
+      if: type = pull_request
+      language: node_js
+      node_js: "node"
+      install:
+        - pyenv global 2.7.15
+        - make doc-only
+        - npm install -g linkinator
+      script:
+        - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
+            DOC_FILES=`curl -sL https://github.com/nodejs/node/pull/${TRAVIS_PULL_REQUEST}.patch | grep -o '\bdoc/api/.*\.md\b'`;
```

```suggestion
            DOC_FILES=`git diff --name-only HEAD...$TRAVIS_BRANCH | grep -o '\bdoc/api/.*\.md\b'`;
```

Refs: https://stackoverflow.com/questions/41145041/list-files-modified-in-a-pull-request-within-travis

---

I think we had trouble with commit message linting before with `$TRAVIS_BRANCH` which is why we ended up downloading the patch files but I forget the details.